feat(angular): provide angular standalone components (#1325)
siemens/ix: BREAKING_CHANGES.md

# Breaking Changes

Welcome to the comprehensive list of breaking changes introduced in `Siemens Industrial Experience`.
This document aims to provide a clear and detailed overview of all significant modifications that may affect your existing codebase. Understanding these changes is crucial for a smooth transition and to ensure compatibility with the latest version of the library.

## Versions

Please select your target version

- [Version v3.0.0](./BREAKING_CHANGES/v3.md)
- [Version v2.0.0](./BREAKING_CHANGES/v2.md)

## Questions ❓🙋‍♀️

> If you have further questions or you are facing any problems during migration please [contact us over our forum](https://community.siemens.com/c/ix/).

Reviewer: ```suggestion
If you have any further questions or you encounter any issues during the migration, please [contact us over our forum](https://community.siemens.com/c/ix/).
```
